websites/H/HBO Max/presence.ts

```diff
@@ -190,7 +215,8 @@ presence.on("UpdateData", async () => {
 					startTimestamp: timestamps[0],
 					endTimestamp: timestamps[1],
 				});
-			}
+			} else delete presenceData.endTimestamp;
```

Comment:
I'm pretty sure this is useless since endTimestamp isn't set anywhere before this